regulations/generator/sidebar/analyses.py

```diff
@@ -42,10 +42,11 @@ def fetch_data(self, http_client):
         """Retrieves SxSEntry data from the server"""
         if 'Interp' in self.label_id:
             reg_part = self.label_parts[0]
-            data = http_client.layer(
-                'analyses', reg_part + '-Interp', self.version)
+            doc_id = '{}/{}-Interp'.format(self.version, reg_part)
+            data = http_client.layer('analyses', 'cfr', doc_id)
```

Comment:
Also minor: we use a lot of different methods to build strings throughout this patch (str.format, sprintf-style formatting with %, str.join, and old fashioned string concatenation). This doesn't affect functionality at all but makes the diffs like 1% harder to read.